recipes/mplhep_data/meta.yaml

```diff
@@ -0,0 +1,38 @@
+{% set name = "mplhep_data" %}
+{% set version = "0.0.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 82915932f0c0aa35aef6a96c54a767d4f86ffdee42b0f23b7be6a872e8141fc5
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python >=3.7
+    - pip
+  run:
+    - python >=3.7
+
+test:
+  imports:
+    - mplhep_data
+
+about:
+  home: "https://github.com/scikit-hep/mplhep_data"
+  license: MIT
+  license_file: LICENSE
```

Comment:
I haven't checked but the other licenses likely require that the text is included. This can either be concatenated into a single file or the `license_file` field can be a list